### DISCUSSION TITLE
[Application] AvatarMetaverse

### DISCUSSION THREAD

**Discussion by ZhangPeibin** [filecoin-project/community#370] [Project Showcase]
Next is the time of the Metaverse!
No matter where they are in Metaverse or wherever they are, the characters are there and have a strong personalization!
Our goal is to provide everyone with an Avatar in Metaverse !

We will use NFT.storage to store and produce metadata for everyone’s avatar, and finally store it on the polygon chain ！
Eventually we will implement a protocol of DID + Wallet address = Avatar NFT

**Comment by TimGuo7** [comment]
great, please show more details about it
